src/content/peerconnection/restart-ice/js/main.js

```diff
@@ -32,6 +32,8 @@ remoteVideo.addEventListener('loadedmetadata', function() {
   console.log(`Remote video videoWidth: ${this.videoWidth}px,  videoHeight: ${this.videoHeight}px`);
 });
 
+const useTransportIceEvent = window.RTCIceTransport && 'onselectedcandidatepairchange' in RTCIceTransport.prototype;
```

Comment:
There are many ICE events (I think).
What about calling it "useTransportCandidatePairChange"? Verbose, but accurate.

Reply:
useSelectedCandidatePairChange